services-js/commissions-app/src/client/common/TextInput.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export interface Props {
+  name: string;
+  placeholder: string;
+  required: boolean;
+}
+export default class FormWithElement extends React.Component<Props> {
+  render() {
+    return (
+      <div className="fs">
+        <div className="fs-c">
+          <div className="txt m-b300">
+            <label
+              htmlFor="FeedbackForm-${this.props.name}"
+              className="txt-l txt-l--sm"
+            >
+              {this.props.name}
```

Comment:
Remember that `name` is a form `<input>` name, so it’ll be something like `firstName`. You’ll want a different prop for the label.

Reply:
So something like this?         --->      htmlFor="FeedbackForm-${this.props.firstName}"

Reply:
This is about the text within the `<label>`. The `htmlFor` is totally fine as it is.